### ISSUE TITLE
ClassificationAccuracyContext: Unable to compare float predictions and integer ground truth

### ISSUE THREAD

**Issue by seraphimstreets** [intel/dffml#1390]
There is a bug regarding the ClassificationAccuracy scorer, where if a model outputs a float value as its classification (eg. 4.0 instead of 4) , it cannot be compared with the integer ground truth and returns 0.0 accuracy. I encountered this problem when using the example given in XGBClassifier model (Iris dataset). Should the scorer auto-cast predictions to integer, or is this the responsibility of the plugin contributor?

## How to reproduce

Basically just the example given in model/xgboost/dffml_model_xgboost/xgbclassifier.py

## Expected behavior

When I modified the scorer to cast to integer, I got near 1.0 or at 1.0 accuracy. 

![image](https://user-images.githubusercontent.com/51947676/171825410-005a5554-2479-4394-9afe-df11aebf1ef5.png)
